assets/community/shaping.toml

```
@@ -26,4 +26,264 @@
+regex = "553 5\\.7\\.1 \\[BL"
+trigger = {Threshold="5/hr"}
+action = "Suspend"
+duration = "2 hours"
+
+# 421 Max message per connection reached, closing transmission channel
+[["yahoo.com".automation]]
+regex = "Max message per connection reached, closing transmission channel"
+trigger = {Threshold="3/hr"}
+action = {SetConfig={name="max_message_rate", value="10/minute"}}
+duration = "1 hour"
+
+#===== web.de
+["web.de"]
+max_deliveries_per_connection = 100
+connection_limit = 4
+enable_tls = "Required"
+consecutive_connection_failures_before_delay = 5
+
+[["web.de".automation]]
+regex = "Too many connections"
+trigger = {Threshold="3/hr"}
+action = {SetConfig={name="connection_limit", value=1}}
+duration = "2 hours"
+
+[["web.de".automation]]
+regex = "Reject due to policy restrictions."
+trigger = {Threshold="5/hr"}
+action = {SetConfig={name="max_message_rate", value="1/minute"}}
+duration = "1 hour"
+
+[["web.de".automation]]
+regex = "Reject due to policy violations."
+trigger = {Threshold="5/hr"}
+action = "Suspend"
+duration = "1 hour"
+
+[["web.de".automation]]
+regex = "554.*Reject due to policy restrictions."
+trigger = {Threshold="5/hr"}
+action = "Suspend"
+duration = "2 hours"
+
+#===== outlook.com
+[["outlook.com".automation]]
+regex = "Unfortunately.*part of their network is on our block list"
+trigger = {Threshold="5/hr"}
+action = "Suspend"
+duration = "2 hours"
+
+[["outlook.com".automation]]
+regex = "Access denied, please try again later"
+trigger = {Threshold="5/hr"}
+action = "Suspend"
+duration = "1 hour"
+
+[["outlook.com".automation]]
+regex = "Server busy. Please try again later"
+trigger = {Threshold="5/hr"}
+action = "Suspend"
+duration = "30m"
+
+[["outlook.com".automation]]
+regex = "The mail server.*has been temporarily rate limited"
+trigger = {Threshold="3/hr"}
+action = "Suspend"
+duration = "30m"
+
+[["outlook.com".automation]]
+regex = "Server busy. Please try again later"
+trigger = {Threshold="3/hr"}
+action = {SetConfig={name="max_message_rate", value="1/minute"}}
+duration = "15m"
+
+[["outlook.com".automation]]
+regex = "The mail server.*has exceeded the maximum number of connections"
+trigger = {Threshold="2/hr"}
+action = {SetConfig={name="connection_limit", value=1}}
+duration = "1 hour"
+
+[["outlook.com".automation]]
+regex = "The mail server.*has exceeded maximum number of messages per connection"
+trigger = {Threshold="2/hr"}
+action = {SetConfig={name="max_deliveries_per_connection", value=2}}
+duration = "1 hour"
+
+[["outlook.com".automation]]
+regex = "Connection refused from.*due to exceed max concurrent connections. IB007"
+trigger = {Threshold="2/hr"}
+action = {SetConfig={name="connection_limit", value=1}}
+duration = "1 hour"
+
+[["outlook.com".automation]]
+regex = "Temporary server error. Please try again later"
+trigger = {Threshold="5/hr"}
+action = "Suspend"
+duration = "1 hour"
+
+#===== orange.fr
+["orange.fr"]
+max_deliveries_per_connection = 75
+connection_limit = 1
```

```suggestion
connection_limit = 2
```

The postmaster page says: `Sending limits: Orange currently allows up to two simultaneous connections per sending IP address, and up to 100 recipients per message`

---

Thanks, good call. I did not check the postmastery page again.. But usually in my experience orange was very strict on sending too much/fast.